bootstrap-promises/bootstrap.cf

```diff
@@ -0,0 +1,158 @@
+#####################################################################################
```

Comment:
now that it's here it can be called failsafe.cf :)
Moreover I think having a copy names promises.cf may be useful to avoid having agent run to fail.